evm/vm/forks/byzantium/headers.py

```diff
@@ -0,0 +1,78 @@
+from evm.constants import (
+    EMPTY_UNCLE_HASH,
+    DIFFICULTY_ADJUSTMENT_DENOMINATOR,
+    DIFFICULTY_MINIMUM,
+    BOMB_EXPONENTIAL_PERIOD,
+    BOMB_EXPONENTIAL_FREE_PERIODS,
+    BYZANTIUM_DIFFICULTY_ADJUSTMENT_CUTOFF,
+)
+from evm.validation import (
+    validate_gt,
+    validate_header_params_for_configuration,
+)
+from evm.vm.forks.frontier.headers import (
+    create_frontier_header_from_parent,
+)
+
+
+def compute_byzantium_difficulty(parent_header, timestamp):
+    """
+    https://github.com/ethereum/EIPs/issues/100
+    TODO: figure out how to know about uncles in this context...
```

Comment:
Is this still TODO? If so, it'd be nice if you could expand on what exactly you mean

Reply:
This was from before I'd put in the optimization to just check if `parent_header.uncles_hash != EMPTY_UNCLE_HASH`.